web/impact/impact/views/calendar_reminder_view.py

```diff
@@ -0,0 +1,52 @@
+from django.views import View
+from django.http import HttpResponseRedirect
+from add2cal import Add2Cal
+import datetime
+from django.http import (
+    JsonResponse,
+    HttpResponse
+)
+
+DATE_FORMAT = "%Y%m%dT%H%M%S"
+
+
+class CalendarReminderView(View):
+
+    def get(self, request, *args, **kwargs):
+        params = self.request.GET
+        start = params.get('start', datetime.datetime.now().strftime(
+            DATE_FORMAT))
+        end = params.get('end', datetime.datetime.now().strftime(DATE_FORMAT))
+        title = params.get('title', 'new reminder')
+        description = params.get('description', '')
+        location = params.get('location', 'Boston, MA')
```

Comment:
Is this the best default for this, considering that it's incorrect for > 80% of our programs? Something like "MassChallenge" might be better, although that unfortunately violates a previous resolution to avoid hard-coding that string in our code.

Reply:
This is something to discuss with Jeff. I'm going to pass on making this change here since we expect to have a location passed in every time for our purposes. if it becomes an issue we can re-raise and discuss.

Reply:
changed default location to MassChallenge for now